contracts/coordinator/src/contract.rs

```diff
@@ -59,6 +59,12 @@ pub fn execute(
         ExecuteMsg::SetActiveVerifiers { next_worker_set } => {
             execute::set_active_worker_set(deps, info, next_worker_set)
         }
+        ExecuteMsg::SetNextVerifiers { next_worker_set } => {
+            execute::set_next_worker_set(deps, info, next_worker_set)
+        }
+        ExecuteMsg::AddSupportedChainsForWorker { chains, worker } => {
```

Comment:
I think this needs to be access restricted to the service registry. Right now, anyone can call this with whatever input they like. I think this can be removed though and the info can just be inferred from the above two methods (`SetActiveVerifiers` and `SetNextVerifiers`)

Reply:
Removed